src/cgame/cg_hud_editor.c

```diff
@@ -761,7 +762,39 @@ qboolean CG_HudSave(int HUDToDuplicate, int HUDToDelete)
 
 		CG_CloneHud(hud2, hud);
 
-		Q_strncpyz(hud2->name, va("%s_copy", hud->name), sizeof(hud2->name));
+		// determine a new name
+		for (attemptedSpot = 1; attemptedSpot < 100; attemptedSpot++)
```

Comment:
`MAXHUDS` is 32 so this doesn't really need to go this high (and better to use a constant here too rather than a magic number)

https://github.com/etlegacy/etlegacy/blob/1827e3a5618ee2b65b96aa84f32b841d2dac438e/src/cgame/cg_local.h#L4384

Reply:
No magic number! :heart:

Reply:
Done